### PR TITLE
O365 integration

### PR DESCRIPTION
**Pull request by tbsbdr**
Added preliminary docs for the Office 365 integration
@jnweiger could you have a look if there is something missing as you just walked through the process
@mmattel please check if the docs is comprehensible and useful for users

### CODE REVIEW

**Comment by mmattel** [comment]
Looks fine to me, but waiting on the fnal ok from @jnweiger 

Backport to 10.9 and 10.8 necessary

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:65]:
```suggestion
# Use `None` if you are using OpenID Connect.
```

**Review by phil-davis — APPROVED** [review]
LGTM - @jnweiger OK?

**Review by mmattel — DISMISSED** [review]

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc
Comment (jnweiger) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:60]: curl https://wopi.owncloud.com/wopi/files/
curl: (6) Could not resolve host: wopi.owncloud.com

??? should that work?

**Review by jnweiger — DISMISSED** [review]
Still cannot confirm the correctness of the proxy setup. Sorry.
I only have a working test setup without proxy.

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc
Comment (mmattel) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:50]: I have updated the link definition so it will not get a clickable pointing to nowhere

**Comment by mmattel** [comment]
Missing the key from https://github.com/owncloud/wopi/issues/195 (wopi.business-flow.enabled)

**Comment by mmattel** [comment]
The key `wopi.business-flow.enabled` is now added in the config sample description, see: https://github.com/owncloud/docs-server/pull/263 (Add WOPI Business Key)

**Comment by mmattel** [comment]
@tbsbdr I have updated the PR as we discussed. Please take a look and approve if ok.

**Comment by tbsbdr** [comment]
@mmattel LGTM, thx

**Pull request merged**
